tests/foreman/cli/test_myaccount.py

```diff
@@ -165,23 +159,64 @@ def test_negative_update_email(self):
         for email in invalid_emails_list():
             with self.subTest(email):
                 with self.assertRaises(CLIReturnCodeError):
-                    User.update({
-                        'login': self.user['login'],
-                        'mail': email,
-                    })
+                    self.update_user(
+                        {'login': self.user['login'], 'mail': email}
+                    )
 
-    @stubbed()
     @tier1
-    def test_negative_update_password_invalid(self):
-        """Update My Account with invalid Password/Verify fields
+    def test_negative_update_locale(self):
+        """Update My Account with invalid Locale
 
-        @id: f9230699-fb8e-45d6-a0c2-abb8b751304d
-
-        @Steps:
-        1. Update Current user with all variations of Password/Verify fields
-        in [2]
+        @id: 6f63c9b4-80e6-11e6-8ea1-68f72889dc7f
 
         @Assert: User is not updated. Appropriate error shown.
+        """
+        with self.assertRaises(CLIReturnCodeError):
+            self.update_user({'locale': 'invalid'})
 
-        @caseautomation: notautomated
+
+class MyAccountEphemeralUserTestCase(CLITestCase):
```

Comment:
Code uses test user credentials to access hammer. So on password case I would need reverting the changes on test user, risking some race condition. I commented that on setUpClass, maybe it would be better commenting on class. I am going to check performance impact with all tests having their user.